CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - `tidy3d convert` from `.lsf` files to tidy3d scripts is moved to another repository at `https://github.com/hirako22/Lumerical-to-Tidy3D-Converter`.
+- Relax validation of smallest triangles area in `TriangleMesh` from 1e-7 to 1e-36.
```

Comment:
minor comment: consider leaving out the specific numbers or putting in single backslash. (also I guess units of um^2?)